docs/rest.md

```diff
@@ -222,9 +226,10 @@ Sample result:
 
 #### Fields
 * `snaps`
-    * `status`: can be either `not installed`, `installed`, `active` (i.e. is
-      current), `removed` (but data present); there is no `purged` state, as a
-      purged snap is undistinguishable from a non-installed snap.
+    * `status`: can be either `buyable`, `available`, `installed`,
```

Comment:
Why is buyable “such a terrible term”? It's synonymous with “for sale” or “purchasable”, and you're asking that we use “buy” as the verb of the action that moves the snap from this state to available. It seems a natural fit.